src/components/Player/Header/PlayerBadges.jsx

```diff
@@ -171,42 +182,41 @@ export const PlayerBadgesIcons = ({
               <IconSteam className="icon" />
             </a>
           </div>
-          {Math.round(new Date().getTime() / 1000.0) >= Number(tracked) ? (
+          {cheese > 0 && (
             <div
-              className="iconButton iconEye"
-              data-hint={strings.app_untracked}
+              className="iconButton"
+              data-hint={`${cheese} ${strings.app_cheese_bought}`}
               data-hint-position="top"
             >
-              <IconEyeInactive className="icon" />
+              <IconCheese className="cheese icon" />
             </div>
-          ) : (
+          )}
+          {isSubscriber && (
             <div
-              className="iconButton iconEye"
-              data-hint={strings.app_tracked}
+              className="iconButton iconContributor"
+              data-hint={`${strings.app_subscriber}`}
               data-hint-position="top"
             >
-              <IconEye className="iconEyeTracked" />
+              <IconContributor
```

Comment:
Are we reusing the same icon for subscribers and contributors?

Reply:
yes differtent color though